dailizabot/text_patterns.py

```diff
@@ -13,4 +13,18 @@
     "Würde {0} dir denn wirklich helfen?",
     "Bist du sicher, dass du {0} brauchst?"]],
 
+    [r"Ich verstehe (.*) nicht",
```

Comment:
```suggestion
    [r"Ich verstehe? (.*) nicht",
```
added ? to regex expression

Reply:
Good idea. Thank you!